conda_forge_tick/migrators/migration_yaml.py

```diff
@@ -285,10 +285,11 @@ def migrate(
 
     def pr_body(self, feedstock_ctx: ClonedFeedstockContext) -> str:
         body = super().pr_body(feedstock_ctx)
+        url = f"https://conda-forge.org/status/migration/?name={self.name}"
```

Comment:
It is weird, but you need to use `get_migrator_name` from `conda_forge_tick.utils` here. I should clean this up in another PR.

Reply:
@beckermr I have made this change (hopefully correctly).